### PR TITLE
Switch to manala

### LINKED ISSUE #1: Move from ElaoInfra/* to Manala/*

### PR DESCRIPTION
**Pull request by chalasr**
Closes #1

### CODE REVIEW

**Comment by nervo** [comment]
Why aliasing Application to Manala in https://github.com/manala/manalize/blob/manala/bin/manalize#L15 ?
Keeping "Application" would make it a little more name agnostic, no ?

**Comment by chalasr** [comment]
No real reason, let's use its original name 👍
This class will be useless as soon as symfony 3.2 is released (see http://symfony.com/blog/new-in-symfony-3-2-console-improvements-part-1#better-support-for-one-command-applications)

**Comment by nervo** [comment]
Ah, nice ! Do you plan to have some kind of todo list somewhere ?

**Comment by chalasr** [comment]
See https://github.com/manala/manalize/issues/10 :) Feel free to discuss about some points, update them or add new ones

File: src/Resources/ansible/group_vars/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+##########
+# Shared #
+##########
+
+manala_deploy_shared_files:
+  - app/config/parameters.yml
+
+manala_deploy_shared_dirs:
+  - var/logs
+  - var/geoip2
```
Comment (benji07) [inline src/Resources/ansible/group_vars/deploy.yml]: faudrait enlever ces exemples :)

Comment (chalasr) [reply src/Resources/ansible/group_vars/deploy.yml]: Je vais clean
(english would be greatly appreciated, c'est de l'open source 😆 )

**Pull request merged**